pkgs/applications/audio/lmms/default.nix

```
@@ -70,7 +70,7 @@ mkDerivation rec {
     })
   ];
 
-  cmakeFlags = [ "-DWANT_QT5=ON" ];
+  cmakeFlags = [ "-DWANT_QT5=ON" "-DCMAKE_POLICY_VERSION_MINIMUM=3.5" ];
```

```suggestion
  cmakeFlags = [
    "-DWANT_QT5=ON"
    "-DCMAKE_POLICY_VERSION_MINIMUM=3.5"
  ];
```
the formatter wants that